app/controllers/collections_controller.rb

```diff
@@ -44,7 +53,7 @@ def index
       params[:sort_column] = "collections.created_at" if !valid_sort_column(params[:sort_column], 'collection')
       params[:sort_direction] = 'DESC' if !valid_sort_direction(params[:sort_direction])
       sort = params[:sort_column] + " " + params[:sort_direction]
-      @collections = Collection.sorted_and_filtered(sort, params[:collection_filters], params[:page])
+      @collections = Collection.for_blurb.sorted_and_filtered(sort, params[:collection_filters], params[:page])
```

Comment:
This scope shouldn't be here, it's already in `sorted_and_filtered`.

```suggestion
      @collections = Collection.sorted_and_filtered(sort, params[:collection_filters], params[:page])
```